<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>700</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Calculadora IMC</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>609</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>609</width>
        <height>700</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(112, 112, 112);</string>
       </property>
       <property name="text">
        <string>IMC</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>118</width>
         <height>3</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(112, 112, 112);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>591</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>IMC é a sigla para Índice de Massa Corpórea, parâmetro adotado pela Organização Mundial de Saúde </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>251</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>para calcular o peso ideal de cada pessoa.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(31, 94, 175);</string>
       </property>
       <property name="text">
        <string>Cálculo IMC</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>571</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>O índice é calculado da seguinte maneira: divide-se o peso do paciente pela sua altura elevada ao</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>591</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>quadrado. Diz-se que o indivíduo tem peso normal quando o resultado do IMC está entre 18,5 e 24,9.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>591</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Quer descobrir seu IMC? Insira seu peso e sua altura nos campos abaixo e compare com os indices da</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>421</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>tabela. Importante: siga os exemplos e use pontos como separadores.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(31,94,175,255);</string>
       </property>
       <property name="text">
        <string>Altura</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>65</x>
         <y>231</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(31,94,175,255);</string>
       </property>
       <property name="text">
        <string>(ex.: 1,70)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>230</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(31,94,175,255);</string>
       </property>
       <property name="text">
        <string>Peso</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>215</x>
         <y>231</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(31,94,175,255);</string>
       </property>
       <property name="text">
        <string>(ex.: 69,2)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnCalcular">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">#BtnCalcular{
	background-color:#1f5eaf;
	color: #fff;
 	border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>Calcular</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons8-divisa-para-cima-90 (1).png</normaloff>
         <normalon>icons8-divisa-para-cima-24.png</normalon>icons8-divisa-para-cima-90 (1).png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnLimpar">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>300</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">#BtnLimpar{
	background-color:#ff5202;
	color: #fff;
 	border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>Limpar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons8-divisa-para-cima-90 (1).png</normaloff>
         <normalon>icons8-divisa-para-cima-24.png</normalon>icons8-divisa-para-cima-90 (1).png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="Lfundo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>370</y>
         <width>521</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#Lfundo{
	background-color: #1c5cac;
	color: #fff;
	border-radius:5px;
}</string>
       </property>
       <property name="text">
        <string>    VEJA A INTERPRETAÇÃO DO IMC</string>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>425</y>
         <width>521</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>521</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>400</y>
         <width>31</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #1c5cac;</string>
       </property>
       <property name="text">
        <string>IMC</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>400</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #1c5cac;</string>
       </property>
       <property name="text">
        <string>CLASSIFICAÇÃO</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>400</y>
         <width>71</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #1c5cac;</string>
       </property>
       <property name="text">
        <string>OBESIDADE</string>
       </property>
      </widget>
      <widget class="Line" name="line_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>500</y>
         <width>521</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>540</y>
         <width>521</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>580</y>
         <width>521</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>620</y>
         <width>521</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="LeAltura">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#LeAltura{
	background-color:#fff;
	border-radius:10px;	
	border: 1px solid #5786c3;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Metros</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LePeso">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>250</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">#LePeso{
	border-radius:10px;
	border: 1px solid #5786c3;
}</string>
       </property>
       <property name="placeholderText">
        <string>Quilos</string>
       </property>
      </widget>
      <widget class="QLabel" name="LblMenorque18">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>426</y>
         <width>491</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>MENOR QUE 18,5                      MAGREZA                                          0</string>
       </property>
      </widget>
      <widget class="QLabel" name="LblEntre18e24">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>461</y>
         <width>491</width>
         <height>38</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>ENTRE 18,5 E 24,9                     NORMAL                                           0</string>
       </property>
      </widget>
      <widget class="QLabel" name="LblEntre25e29">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>502</y>
         <width>491</width>
         <height>37</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>ENTRE 25,0 E 29,9                     SOBREPESO                                      I</string>
       </property>
      </widget>
      <widget class="QLabel" name="LblEntre30e39">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>542</y>
         <width>491</width>
         <height>37</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>ENTRE 30,0 E 39,9                     OBSIDADE                                        II</string>
       </property>
      </widget>
      <widget class="QLabel" name="LblMaiorque40">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>582</y>
         <width>491</width>
         <height>37</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>MAIOR QUE 40,0                       OBSIDADE GRAVE                              III</string>
       </property>
      </widget>
      <widget class="QLabel" name="imcres">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>630</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#imcres{
	background-color:#ff5202;
	color: #fff;
 	border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>640</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>SEU IMC:</string>
       </property>
      </widget>
      <widget class="QLabel" name="LblResposta">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>660</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>400</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #1c5cac;</string>
       </property>
       <property name="text">
        <string>(GRAU)</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
